### PR TITLE
5830: Go-To menu shows up blank except for address bar

### PR DESCRIPTION
**Pull request by ElderOrb**
Fix for https://highfidelity.fogbugz.com/f/cases/5830/Go-To-menu-shows-up-blank-except-for-address-bar

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2675/HighFidelity-Beta-PR11231-7f46a06b08d36e2a77c706bd5b6ee2331be17cde.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2675/HighFidelity-Beta-PR11231-7f46a06b08d36e2a77c706bd5b6ee2331be17cde.exe

**Comment by cain-kilgore** [comment]
Hey @ElderOrb: Is there a  Test Plan for this?

**Comment by ElderOrb** [comment]
Unfortunately I have nothing to add to what Anshuman said: 

> I was able to reproduce this bug in 6988 by opening and closing the GoTo menu repetitively. Although it's not common, it seems to happen more consistently right after start up. I've attached logs from my repro today.

Oh, yes, this is specific to tablet version of menu ONLY (no matter VR or 2D)

**Comment by cain-kilgore** [comment]
Okay, I read the FB Notes - I'll give this a go now and see the results.

File: interface/resources/qml/hifi/tablet/TabletAddressDialog.qml [hunk 1]
```diff
@@ -39,11 +39,25 @@ StackView {
...
     function rpc(method, parameters, callback) {
+        console.debug('TabletAddressDialog: rpc: method = ', method, 'parameters = ', parameters, 'callback = ', callback)
+
         rpcCalls[rpcCounter] = callback;
         var message = {method: method, params: parameters, id: rpcCounter++, jsonrpc: "2.0"};
         sendToScript(message);
     }
     function fromScript(message) {
+        if(message.method == 'refreshFeeds')
+        {
```
Comment (zfox23) [inline interface/resources/qml/hifi/tablet/TabletAddressDialog.qml]: Tiny styling and logic fix - please format this and the above line as:
`if (message.method === 'refreshFeeds') {`

**Comment by cain-kilgore** [comment]
# Test Passed

I wasn't able to reproduce the white screen issue in the Tablet after many, many tries. Both in Desktop and HMD.

All good from my end!

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Comment by ElderOrb** [comment]
> Will be dev-approved after one simple fix!

done

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2691/HighFidelity-Beta-PR11231-9cd784990cbb445a7afb9d1bb4a9ca16c83ee42f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2691/HighFidelity-Beta-PR11231-9cd784990cbb445a7afb9d1bb4a9ca16c83ee42f.exe

**Pull request merged**